wpilibj/src/main/java/edu/wpi/first/wpilibj/kinematics/ChassisSpeeds.java

```diff
@@ -47,36 +47,39 @@ public ChassisSpeeds() {
   /**
    * Constructs a ChassisSpeeds object.
    *
-   * @param vx    Forward velocity.
-   * @param vy    Sideways velocity.
-   * @param omega Angular velocity.
+   * @param vxMetersPerSecond     Forward velocity.
+   * @param vyMetersPerSecond     Sideways velocity.
+   * @param omegaRadiansPerSecond Angular velocity.
    */
-  public ChassisSpeeds(double vx, double vy, double omega) {
-    this.vx = vx;
-    this.vy = vy;
-    this.omega = omega;
+  public ChassisSpeeds(double vxMetersPerSecond, double vyMetersPerSecond,
+                       double omegaRadiansPerSecond) {
+    this.vxMetersPerSecond = vxMetersPerSecond;
+    this.vyMetersPerSecond = vyMetersPerSecond;
+    this.omegaRadiansPerSecond = omegaRadiansPerSecond;
   }
 
   /**
    * Converts a user provided field-relative set of speeds into a robot-relative
    * ChassisSpeeds object.
    *
-   * @param vx         The component of speed in the x direction relative to the field.
-   *                   Positive x is away from your alliance wall.
-   * @param vy         The component of speed in the y direction relative to the field.
-   *                   Positive y is to your left when standing behind the alliance wall.
-   * @param omega      The angular rate of the robot.
-   * @param robotAngle The angle of the robot as measured by a gyroscope. The robot's angle
-   *                   is considered to be zero when it is facing directly away from your
-   *                   alliance station wall. Remember that this should be CCW positive.
+   * @param vxMetersPerSecond     The component of speed in the x direction relative to the field.
+   *                              Positive x is away from your alliance wall.
+   * @param vyMetersPerSecond     The component of speed in the y direction relative to the field.
+   *                              Positive y is to your left when standing behind the alliance wall.
+   * @param omegaRadiansPerSecond The angular rate of the robot.
+   * @param robotAngle            The angle of the robot as measured by a gyroscope. The robot's
```

Comment:
Should specify what units angle is in.  Degrees?  Radians?

Reply:
It's a `Rotation2d`. Users can use `Rotation2d.fromDegrees()` for degrees or the normal `Rotation2d` constructor for radians. These are documented in the `Rotation2d` class.